experimental/CliffordAlgOrd/src/IO.jl

```diff
@@ -0,0 +1,73 @@
+##################################################
+#
+#  Clifford algebra and elements
+#
+##################################################
+ 
+##### Algebra #####
+function Base.show(io::IO, ::MIME"text/plain", C::CliffordAlgebra)
+  io = pretty(io)
+  print(io, "Clifford algebra of quadratic space with Gram matrix\n")
```

Comment:
```suggestion
  print(io, LowercaseOff(), "Clifford algebra of quadratic space with Gram matrix\n")
```
since "Clifford" is a name, we always want to print it uppercase. This prevents other code from lowercasing it.